packages/perseus-editor/src/widgets/expression-editor.tsx

```diff
@@ -509,11 +584,14 @@ class AnswerOption extends React.Component<
     };
 
     toggleConsidered = () => {
+        // Update findNextIn to have stronger typing.
         const newVal = findNextIn(
             PerseusExpressionAnswerFormConsidered,
             this.props.considered,
         );
-        this.change({considered: newVal});
+        this.props.onChangeConsidered(
+            newVal as (typeof PerseusExpressionAnswerFormConsidered)[number],
```

Comment:
As a general rule, we should avoid `as` as much as possible. `as` tells TypeScript that we know what the type should be and suppresses type checking. If you can use `satisfies` that'd be better. Finally, if the `findNExtIn()` adjustment in my previous comment works, you won't need this type "cast" at all.